runtime/src/tasks/qthreads/tasks-qthreads-launch.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// Launch assistance for the qthreads tasking layer.
+//
+
+#define _POSIX_C_SOURCE 200112L  // for setenv(3) in <stdlib.h>
+
+#include <assert.h>
+#include <stdlib.h>
+
+#include "chplrt.h"
+#include "chpl-tasks-launch.h"
+#include "chpl-env.h"
+#include "error.h"
+
+
+void chpl_task_preLaunch(void) {
```

Comment:
I don't think we need to do this before launching. I would expect the SPINCOUNT setting to go in `setupSpinWaiting()` in the qthreads shim (and I think I would make a `setupAffinity()` to go with it)

I think that's the natural place for it, and it also allows us to use our `chpl_qt_setenv()` helper which does the right thing since there are `QT_` and `QTHREAD_` variants of all env vars.